pages/about.html

```diff
@@ -1,12 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Me</title>
+    <link rel="stylesheet" href="../styles/styles.css">
+  </head>
   
-</body>
+  <body class="wallpaper">
+    <header>
+      <nav class="page-links">
+        <a href="/Users/nancylee/Developer/projects/personal-portfolio-site/pages/index.html">Home</a>
+        <a href="/Users/nancylee/Developer/projects/personal-portfolio-site/pages/about.html">About Me</a>
+        <a href="/Users/nancylee/Developer/projects/personal-portfolio-site/pages/portfolio.html">Portfolio</a>
+      </nav>
+    </header>
+    <article>
+      <h1>This is Nancy...</h1>
+      <p>I enjoy eating, sleeping and small gatherings with friends. Here you will find a picture of my dream kitchen, 
+        Tenoch Huerta representing Mexican Indigenous people in the movie Black Panther: Wakanda Forever, 
+        and various traditional wear from Mexican culture.</p>
+    </article>
+      <div id="favorites">
+        <img src="/Users/nancylee/Developer/projects/personal-portfolio-site/images/Kitchen.jpg" alt="kitchen">
```

Comment:
It would be great if the alternate text could be more descriptive. People who are blind or have low vision use screen readers to tell them what's on the page. Screen readers use the alt text to describe the picture to the user. Just hearing "kitchen" doesn't give the full context of the picture for the user.